manifests/init.pp

```diff
@@ -56,23 +56,17 @@
 #   }
 #
 class nsclient (
-  $allowed_hosts           = $nsclient::params::allowed_hosts,
-  $service_state           = $nsclient::params::service_state,
-  $service_enable          = $nsclient::params::service_enable,
-  $package_source_location = $nsclient::params::package_source_location,
-  $package_source          = $nsclient::params::package_source,
-  $package_name            = $nsclient::params::package_name,
-  $download_destination    = $nsclient::params::download_destination,
-  $config_template         = $nsclient::params::config_template,
-  $install_path            = $nsclient::params::install_path
+  $allowed_hosts                  = $nsclient::params::allowed_hosts,
+  $service_state                  = $nsclient::params::service_state,
+  $service_enable                 = $nsclient::params::service_enable,
+  String $package_source_location = $nsclient::params::package_source_location,
+  String $package_source          = $nsclient::params::package_source,
+  String $package_name            = $nsclient::params::package_name,
+  $download_destination           = $nsclient::params::download_destination,
+  String $config_template         = $nsclient::params::config_template,
+  String $install_path            = $nsclient::params::install_path
```

Comment:
Not absolute path?

Reply:
I always try to only replace the old validate method. In this case it was validate_string and not validate_path